examples/demo/pingpong/src/pingpong/j/model/associations/LeftPlayer.java

```diff
@@ -0,0 +1,13 @@
+package pingpong.j.model.associations;
+
+import hu.elte.txtuml.api.model.Composition;
+import pingpong.j.model.Player;
+import pingpong.j.model.Game;
+
+public class LeftPlayer extends Composition {
+	public class inGame extends HiddenContainer<Game> {
```

Comment:
Why isn't it capital? Same true for player and PlayerOwnsRacket.player and .racket and for RightPlayer.